src/main/java/com/wavefront/opentracing/WavefrontTracer.java

```diff
@@ -83,6 +83,9 @@
   private final static String DURATION_SUFFIX = ".duration.micros";
   private final static String OPERATION_NAME_TAG = "operationName";
 
+  private final static String SPAN_KIND_TAG_KEY = "span.kind";
+  private final static String SPAN_KIND_TAG_VALUE_DEFAULT = "unknown";
```

Comment:
@sushantdewan123 should we default to "none" (`NULL_TAG_VAL`) to be consistent with application tags?

Reply:
Changed to using `NULL_TAG_VAL` for now unless we have other concerns on this.